omnibus/config/templates/supermarket-ctl/supermarket-ctl.erb

```diff
@@ -15,20 +16,32 @@
 # limitations under the License.
 #
 
-# Ensure the calling environment (disapproval look Bundler) does not infect our
-# Ruby environment if called from a Ruby script.
-for ruby_env_var in RUBYOPT \
-                    BUNDLE_BIN_PATH \
-                    BUNDLE_GEMFILE \
-                    GEM_PATH \
-                    GEM_ROOT \
-                    GEM_HOME
-do
-  unset $ruby_env_var
-done
-
-# This bumps the default svwait timeout from 7 seconds to 30 seconds
-# As documented at http://smarden.org/runit/sv.8.html
-export SVWAIT=30
-
-<%= embedded_bin %>/omnibus-ctl supermarket <%= embedded_service %>/omnibus-ctl $@
+require "omnibus-ctl"
+
+module Omnibus
+  # This is coded to override omnibus-ctl 0.3.6, which is wayyyyy behind master, but
+  # thats what Supermarket is pinned to (via omnibus-ctl software definition)
+  class SupermarketCtl < Omnibus::Ctl
+
+    # Need to allow '--accept-license' to be passed to the reconfigure command
+    def initialize(name, service_commands=true)
+      super
+      @category_command_map["general"]["reconfigure"][:arity] = 2
+    end
+
+    # Override reconfigure to first perform license-acceptance flow
+    def reconfigure(*args)
+      require "license_acceptance/acceptor"
+      LicenseAcceptance::Acceptor.check_and_persist!("supermarket", "<%= version %>")
```

Comment:
Because some of the `supermarket-ctl` commands use other licensed things—`reconfigure` uses chef-zero, `test` uses InSpec—should the acceptance for Supermarket include those things, too?

Reply:
I definitely still like bringing the `supermarket-ctl` command up to date to Ruby Glory.

Reply:
I think this changes to ...?

```suggestion
      LicenseAcceptance::Acceptor.check_and_persist!(provided: "accept-no-persist")
```